ui/admin/app/styles/_notify.scss

```diff
@@ -6,60 +6,55 @@
 // This file is essentially a rip of the default ember notify styles with some minor tweaks for positioning.
 @-webkit-keyframes ember-notify-show {
   from {
-    left: -310px;
+    right: -310px;
   }
 
   to {
-    left: 0;
+    right: 0;
   }
 }
 @keyframes ember-notify-show {
   from {
-    left: -310px;
+    right: -310px;
   }
 
   to {
-    left: 0;
+    right: 0;
   }
 }
 @-webkit-keyframes ember-notify-hide {
   from {
-    left: 0;
+    right: 0;
   }
 
   to {
-    left: -310px;
+    right: -310px;
   }
 }
 @keyframes ember-notify-hide {
   from {
-    left: 0;
+    right: 0;
   }
 
   to {
-    left: -310px;
+    right: -310px;
   }
 }
 
 .ember-notify-cn {
   bottom: 0;
-  left: 0;
-  max-width: 300px;
+  right: 0;
   position: fixed;
   z-index: 5000;
+  margin-bottom: 16px;
 }
 
 .ember-notify {
   box-sizing: border-box;
   display: block;
-  margin: 10px;
+  margin: 16px 24px;
   position: relative;
-  width: 300px;
-
-  .close {
-    cursor: pointer;
-    float: right;
-  }
+  width: 360px;
```

Comment:
hds toast size is 360px. 
Also this file was using only pixels. Should this be refactored to use rem instead ?

Reply:
Good question, hds uses px.
Since this PR has lots of file changes, Im going to work on converting the _notify.scss to use rem in a separate PR.